src/main/java/seedu/edrecord/ui/AssignmentListCard.java

```diff
@@ -64,7 +64,7 @@
     private TableView<Map.Entry<Assignment, Grade>> assignmentsTable;
 
     /**
-     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     * Creates a {@code AssignmentListCard} with the given {@code Person} and index to display.
```

Comment:
Minor change to the comments
```suggestion
     * Creates an {@code AssignmentListCard} with the given {@code Person}, index and {@code module} to display.
```